README.md

```diff
@@ -113,9 +113,16 @@ sc = SparkContext(conf=conf)
     2>/dev/null
 ```
 
+## Connecting to the Jupyter Notebook
 
+### Tunneling to the Jupyter Notebook
+```bash
+ssh -f -N -L localhost:44041:10.58.2.201:4041 tgu@login1.berkeley.kbase.us
```

Comment:
```suggestion
ssh -f -N -L localhost:44041:10.58.2.201:4041 <kbase_developer_username>@login1.berkeley.kbase.us
```
... where `kbase_developer_username` is an Argonne account starting with `ac.`

hah, the suggestion box can't handle triple backticks

Reply:
👍